tests/infra/remote.py

```diff
@@ -881,6 +881,10 @@ def __init__(
         env = {}
         if enclave_type == "virtual":
             env["UBSAN_OPTIONS"] = "print_stacktrace=1"
+            security_policy_key = "SECURITY_POLICY"
+            if security_policy_key not in os.environ:
+                raise ValueError(f"Security policy environment variable {security_policy_key} is not set")
+            env["SECURITY_POLICY"] = os.environ[security_policy_key]
```

Comment:
`os.environ.get(security_policy_key)` so that it doesn't throw if it's unset?

Reply:
Yeah this was pre the discussion about allowing it to be unset